src/models/ScoreLog.php

```diff
@@ -69,28 +75,49 @@ private static function scorelogFromRow(Map<string, string> $row): ScoreLog {
     int $level_id,
     int $team_id,
     bool $any_team,
+    bool $refresh = false,
   ): Awaitable<bool> {
     $db = await self::genDb();
-
-    if ($any_team) {
-      $result =
-        await $db->queryf(
-          'SELECT COUNT(*) FROM scores_log WHERE level_id = %d AND team_id IN (SELECT id FROM teams WHERE id != %d AND visible = 1)',
-          $level_id,
-          $team_id,
-        );
-    } else {
-      $result =
-        await $db->queryf(
-          'SELECT COUNT(*) FROM scores_log WHERE level_id = %d AND team_id = %d',
-          $level_id,
-          $team_id,
-        );
+    $mc_result = self::getMCRecords('LEVEL_CAPTURES');
+    if ((!$mc_result) || (count($mc_result) === 0) || ($refresh)) {
```

Comment:
I can understand the parens around the middle conditional (even if it isn't necessary), but I'm not a fan of the parens around the first and last conditional.